hypertrace-metrics-generator/hypertrace-metrics-generator/src/main/java/org/hypertrace/metrics/generator/OtlpMetricsSerde.java

```diff
@@ -0,0 +1,55 @@
+package org.hypertrace.metrics.generator;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import io.opentelemetry.proto.metrics.v1.ResourceMetrics;
+import java.util.Map;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OtlpMetricsSerde implements Serde<ResourceMetrics> {
+  private static final Logger LOGGER = LoggerFactory.getLogger(OtlpMetricsSerde.class);
+
+  @Override
+  public void configure(Map<String, ?> configs, boolean isKey) {}
+
+  @Override
+  public void close() {}
+
+  @Override
+  public Serializer<ResourceMetrics> serializer() {
+    return new OtlpMetricsSerde.Ser();
+  }
+
+  @Override
+  public Deserializer<ResourceMetrics> deserializer() {
+    return new OtlpMetricsSerde.De();
+  }
+
+  public static class Ser implements Serializer<ResourceMetrics> {
```

Comment:
Should use real class names if making these public (and make them private if possible)

Reply:
will do this.

Reply:
Done.